docs/entity_registry_index.md

```diff
@@ -9,14 +9,28 @@ Being registered has the advantage that the same entity will always get the same
 
 A user is also able to override the name of an entity in the entity registry. When set, the name of the entity registry is used in favor of the name the device might give itself.
 
-## Unique ID requirements
+## Unique ID
+
+It is important that it is not possible for the user to change the unique ID, because they system would lose all its settings related to it.
+
+An entity is looked up in the registry based on a combination of the platform type (e.g., `light`), and the integration name (domain) (e.g. hue) and the unique ID of the entity. Entities should not include the `domain` (e.g., `your_integration`) and platform type (e.g., `light`) in their Unique ID as the system already accounts for these identifiers.
+
+If a device has a single unique id but provides multiple entities, combine the unique id with unique identifiers for the entities. For example, if a device measures both temperature and humidity, you can uniquely identify the entities using `{unique_id}-{sensor_type}`.
 
-An entity is looked up in the registry based on a combination of the platform type (e.g., `light`), and the integration name (domain) (e.g. hue) and the unique ID of the entity. It is therefore very important that the unique ID is unique! It is also important that it is not possible for the user to change the unique ID, because that means it will lose all its settings related to it.
+## Unique ID requirements
 
-Good sources for a unique ID:
+#### Example acceptable sources for a unique ID
```

Comment:
Should this be level three header or should we change the parent header to level three?

Reply:
Yes